src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationProvider.cs

```diff
@@ -104,7 +84,7 @@ private void Load(bool reload)
         /// </summary>
         /// <exception cref="FileNotFoundException">If Optional is <c>false</c> on the source and a
         /// file does not exist at specified Path.</exception>
-        public override void Load()
+        public override void DoLoad()
```

Comment:
Well the ideal place we end up here is that providers should no longer need to care about errors, the error handling happens outside of provider code... so the answer should eventually be no before we merge :)